### PR TITLE
Use strict

### PR DESCRIPTION
**Pull request by popomore**
Split PR from #383 

Based on #385, merge this PR after that.

### CODE REVIEW

**Comment by jlongster** [comment]
I merged #385 but for some reason this PR isn't updating to show that master already has the single quote commit... Do you know what it hasn't updated? I can't do the code review until I see the single commit for this.

**Comment by popomore** [comment]
I don't know why, I try to rebase for it.

File: tests/globals.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 (function() {
-    var expect, util, lib;
+    'use strict';
+
+    var expect, util, lib, Environment, Loader, templatesPath;
```
Comment (popomore) [inline tests/globals.js]: Should define a variable before use it in strict mode.

**Comment by popomore** [comment]
It can be reviewed now.

**Comment by jlongster** [comment]
Thanks. All tests are passing, so that's good. Let's hope it doesn't break anything else.

**Pull request merged**